release-notes/SupportedFeatures-v2.1.0.md

```diff
@@ -0,0 +1,63 @@
+WCF Features in .Net Core 2.1
+======================================================
+*This is part of [WCF release notes](https://github.com/dotnet/wcf/releases/tag/v2.1.0) for .NET Core 2.1. Please check out release notes for what's new since 1.1.0.*
```

Comment:
Shouldn't this...
`Please check out release notes for what's new since 1.1.0.` 
say "since 2.0.0"  

I would also replace "check out" with "see the".